application/src/main/resources/application-auth.properties

```diff
@@ -1,5 +1,9 @@
 # Profile "auth" is meant to be used in production setting. This is for reference.
 
+# Ressource server
+# spring.security.oauth2.resourceserver.jwt.issuer-uri=http://auth.markus-workstation-starwit/realms/aicockpit
```

Comment:
Please remove your local hostname. Also, the URI should probably be updated like above.